### PR TITLE
Manage reload conf (SIGHUP)

### PR DESCRIPTION
**Pull request by ddurieux**

### CODE REVIEW

File: alignak/daemon.py [hunk 1]
```diff
@@ -333,8 +333,7 @@ def do_mainloop(self):
                 logger.debug('Dumping objects')
                 self.need_objects_dump = False
             if self.need_config_reload:
-                logger.debug('Reloading configuration')
```
Comment (mohierf) [inline alignak/daemon.py]: I think that an INFO (or WARNING?) log should be more appropriate

File: alignak/daemons/arbiterdaemon.py [hunk 2]
```diff
@@ -687,7 +694,7 @@ def run(self):
         logger.debug("Run baby, run...")
         timeout = 1.0
 
-        while self.must_run and not self.interrupted:
+        while self.must_run and not self.interrupted and not self.need_config_reload:
```
Comment (mohierf) [inline alignak/daemons/arbiterdaemon.py]: Where is `self.need_config_reload` initialized?

File: alignak/daemon.py (hunk 1 above)
Comment (ddurieux) [inline alignak/daemon.py]: in  alignak/daemons/arbiterdaemon.py, I have added ```logger.info('Reloading configuration')```

File: alignak/daemons/arbiterdaemon.py (hunk 2 above)
Comment (ddurieux) [inline alignak/daemons/arbiterdaemon.py]: in the ```__init__``` of alignak.daemon.py

File: alignak/daemon.py (hunk 1 above)
Comment (mohierf) [inline alignak/daemon.py]: Ok

File: alignak/daemons/arbiterdaemon.py (hunk 2 above)
Comment (mohierf) [inline alignak/daemons/arbiterdaemon.py]: Ok. I thought it was a new property :wink:

Comment (ddurieux) [inline alignak/daemons/arbiterdaemon.py]: No, this property exist since for very long time :p

**Pull request merged**